### PR TITLE
4 - Hero Video

### LINKED ISSUE #4: Implement Main Hero Video
- [x] Desktop version
- [x] Mobile version
- [x] Text overlay
- [x] CTA Button overlay

### PR DESCRIPTION
**Pull request by andreituicu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #4

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/drafts/tuicu/herovideo
- After: https://4-hero-video--moleculardevices--hlxsites.hlx.page/drafts/tuicu/herovideo

- Before: https://main--moleculardevices--hlxsites.hlx.page
- After: https://4-hero-video--moleculardevices--hlxsites.hlx.page

### CODE REVIEW

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -373,7 +373,8 @@ export async function loadBlock(block) {
           } catch (error) {
             // eslint-disable-next-line no-console
             console.log(`failed to load module for ${blockName}`, error);
-            // console.error(error);
+            // eslint-disable-next-line no-console
+            console.error(error);
```
Comment (mhaack) [inline scripts/lib-franklin.js:377]: Why is this needed? 2 lines above the error is logged.

**Review by mhaack — APPROVED** [review]

File: scripts/lib-franklin.js (hunk 1 above)
Comment (andreituicu) [reply scripts/lib-franklin.js:377]: I thought I had some examples where the log was printing a message, but not the stacktrace. I can't seem to find any now, so maybe I just didn't look close enough. 🙂 I can remove it.
